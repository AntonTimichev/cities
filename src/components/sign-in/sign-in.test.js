import React from "react";
import renderer from "react-test-renderer";
import {MemoryRouter} from 'react-router-dom';

import SignIn from "./sign-in.jsx";

it(`SignIn is rendered correctly`, () => {
  const tree = renderer.create(<MemoryRouter><SignIn /></MemoryRouter>).toJSON();

  expect(tree).toMatchSnapshot();
});
