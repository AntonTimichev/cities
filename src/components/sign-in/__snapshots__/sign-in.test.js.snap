// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignIn is rendered correctly 1`] = `
<main
  className="page__main page__main--login"
>
  <div
    className="page__login-container container"
  >
    <section
      className="login"
    >
      <h1
        className="login__title"
      >
        Sign in
      </h1>
      <form
        action="#"
        className="login__form form"
        method="post"
      >
        <div
          className="login__input-wrapper form__input-wrapper"
        >
          <label
            className="visually-hidden"
          >
            E-mail
          </label>
          <input
            className="login__input form__input"
            name="email"
            onBlur={[MockFunction]}
            placeholder="Email"
            required={true}
            style={
              Object {
                "backgroundColor": "",
              }
            }
            type="email"
          />
        </div>
        <div
          className="login__input-wrapper form__input-wrapper"
        >
          <label
            className="visually-hidden"
          >
            Password
          </label>
          <input
            className="login__input form__input"
            name="password"
            onBlur={[MockFunction]}
            placeholder="Password"
            required={true}
            style={
              Object {
                "backgroundColor": "",
              }
            }
            type="password"
          />
        </div>
        <button
          className="login__submit form__submit button"
          onClick={[Function]}
          type="submit"
        >
          Sign in
        </button>
      </form>
    </section>
    <section
      className="locations locations--login locations--current"
    >
      <div
        className="locations__item"
      >
        <a
          className="locations__item-link"
          href="#"
        >
          <span>
            Amsterdam
          </span>
        </a>
      </div>
    </section>
  </div>
</main>
`;
